<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowOpacity">
   <double>0.000000000000000</double>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>297</width>
       <height>280</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLabel" name="label_message">
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>- message</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_positions">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>x: 0 y:0 z:0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="3">
      <widget class="QToolButton" name="toolButton_draw_3d_2p_line">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/vendor/icons/line.svg</normaloff>:/vendor/icons/line.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QToolButton" name="toolButton_zoom_plus">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../qt_gremlin/icons.qrc">
         <normaloff>:/vendor/icons/zoom_plus.svg</normaloff>:/vendor/icons/zoom_plus.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="12">
      <widget class="QToolButton" name="toolButton_color">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/vendor/icons/format-stroke-color.png</normaloff>:/vendor/icons/format-stroke-color.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="12">
      <widget class="QToolButton" name="toolButton_view_3d">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../qt_gremlin/icons.qrc">
         <normaloff>:/vendor/icons/view-iso.svg</normaloff>:/vendor/icons/view-iso.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QToolButton" name="toolButton_view_front">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../qt_gremlin/icons.qrc">
         <normaloff>:/vendor/icons/view-front.svg</normaloff>:/vendor/icons/view-front.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="11">
      <widget class="QToolButton" name="toolButton_view_right">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../qt_gremlin/icons.qrc">
         <normaloff>:/vendor/icons/view-right.svg</normaloff>:/vendor/icons/view-right.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="10">
      <widget class="QToolButton" name="toolButton_view_left">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../qt_gremlin/icons.qrc">
         <normaloff>:/vendor/icons/view-left.svg</normaloff>:/vendor/icons/view-left.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QToolButton" name="toolButton_view_top">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../qt_gremlin/icons.qrc">
         <normaloff>:/vendor/icons/view-top.svg</normaloff>:/vendor/icons/view-top.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <widget class="QToolButton" name="toolButton_view_bottom">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../qt_gremlin/icons.qrc">
         <normaloff>:/vendor/icons/view-bottom.svg</normaloff>:/vendor/icons/view-bottom.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QToolButton" name="toolButton_view_back">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../qt_gremlin/icons.qrc">
         <normaloff>:/vendor/icons/view-back.svg</normaloff>:/vendor/icons/view-back.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QToolButton" name="toolButton_fit_all">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../qt_gremlin/icons.qrc">
         <normaloff>:/vendor/icons/zoom_extend.svg</normaloff>:/vendor/icons/zoom_extend.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QToolButton" name="toolButton_zoom_min">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../qt_gremlin/icons.qrc">
         <normaloff>:/vendor/icons/zoom_min.svg</normaloff>:/vendor/icons/zoom_min.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_keyboard_input">
       <property name="text">
        <string>keyboard input.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qt_gremlin/icons.qrc"/>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
